troubleshooting.md

```diff
@@ -0,0 +1,570 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+* [Bootstrap VM issues](#bootstrap-vm-related-issues)
+
+* [Following the Installation](#following-the-installation) 
+
+* [Node Related Issues](#node-related-issues)
+
+* [NTP out of sync](#ntp-out-of-sync)
+
+* [Registry Issues](#registry-issues) 
+
+* [IPv6 Disconnected Install Troubleshooting](#ipv6-disconnected-install-troubleshooting)
+
+* [Misc Issues](#misc-issues) 
+
+**Pre-requisites:** 
+
+First and foremost the BM installations follows a general architecture:
+* provisioner node 
+* master node 
+* worker node
+
+Before running the installation, please make sure you have a cleaned the previous files/manifests.
```

Comment:
You don't need any prerequisites. This is a troubleshoot guide so you can remove this.

Reply:
Removed this as suggested.